frontend/src/app/main/ui/workspace/sidebar/options/menus/shadow.cljs

```diff
@@ -152,8 +153,17 @@
     [:div {:class (stl/css-case :global/shadow-option true
                                 :shadow-element true
                                 :dnd-over-top (= (:over dprops) :top)
-                                :dnd-over-bot (= (:over dprops) :bot))
-           :ref dref}
+                                :dnd-over-bot (= (:over dprops) :bot))}
+     (when (some? on-reorder)
+       [:*
+        [:div {:ref dref :class (stl/css :reorder)}
```

Comment:
Q: why the icon needs to be wrapped in a `<div>` ?

Reply:
Because the clickable area is bigger than the icon. This wrapper occupies the 100% of the height and then the icon is positioned in the center of the area.